CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Welcome #
+
+We're so glad you're thinking about contributing to the Mentat project!  If you're unsure or afraid of anything, just ask or submit the issue or pull request anyway.  The worst that can happen is that you'll be politely asked to change something.  We appreciate any sort of contribution, and don't want a wall of rules to get in the way of that.
+
+Before contributing, we encourage you to read our CONTRIBUTING policy (you are here), our [LICENSE](LICENSE), and our [README](README.md), all of which should be in this repository.
```

Comment:
Please omit the clause "all of which should be in this repository". They'll be able to see they're working links.

Reply:
Made the changes